Upstream: Unknown
Reason: automagic cddb dependency
Source: Daniel Mierswa <impulze@impulze.org>

--- configure.ac.orig	2008-09-15 00:50:30.000000000 +0200
+++ configure.ac	2008-09-15 00:50:46.000000000 +0200
@@ -584,7 +584,7 @@
 	[  --enable-cddb           include CDDB lookups in cd_info (default enabled)],	
 	enable_cddb=yes, enable_cddb=no)
 fi
-if test x$enable_cddb != "no" ; then
+if test x$enable_cddb = xyes ; then
    PKG_CHECK_MODULES(CDDB, libcddb >= 1.0.1, [
     HAVE_CDDB=yes 
     AC_DEFINE(HAVE_CDDB, [], [Define this if you have libcddb installed])
